diff --git a/node_modules/react-native-camera/android/build/tmp/compileGeneralDebugJavaWithJavac/previous-compilation-data.bin b/node_modules/react-native-camera/android/build/tmp/compileGeneralDebugJavaWithJavac/previous-compilation-data.bin
new file mode 100644
index 0000000..2ea99ba
Binary files /dev/null and b/node_modules/react-native-camera/android/build/tmp/compileGeneralDebugJavaWithJavac/previous-compilation-data.bin differ
diff --git a/node_modules/react-native-camera/src/RNCamera.js b/node_modules/react-native-camera/src/RNCamera.js
index b7a271a..2df81f6 100644
--- a/node_modules/react-native-camera/src/RNCamera.js
+++ b/node_modules/react-native-camera/src/RNCamera.js
@@ -5,7 +5,6 @@ import {
   findNodeHandle,
   Platform,
   NativeModules,
-  ViewPropTypes,
   requireNativeComponent,
   View,
   ActivityIndicator,
@@ -394,7 +393,6 @@ export default class Camera extends React.Component<PropsType, StateType> {
   };
 
   static propTypes = {
-    ...ViewPropTypes,
     zoom: PropTypes.number,
     useNativeZoom: PropTypes.bool,
     maxZoom: PropTypes.number,
diff --git a/node_modules/react-native-camera/types/index.d.ts b/node_modules/react-native-camera/types/index.d.ts
index 8aa925e..f947cf6 100644
--- a/node_modules/react-native-camera/types/index.d.ts
+++ b/node_modules/react-native-camera/types/index.d.ts
@@ -12,7 +12,7 @@
  * If you are seeing this from the future, please, send us your cutting-edge technology :) (if it exists)
  */
 import { Component, ReactNode } from 'react';
-import { NativeMethods, ViewProperties, findNodeHandle } from 'react-native';
+import { NativeMethods, findNodeHandle } from 'react-native';
 
 type Orientation = Readonly<{
   auto: any;
@@ -483,7 +483,7 @@ export interface HardwareCamera {
 
 export function hasTorch(): Promise<boolean>;
 
-export class RNCamera extends Component<RNCameraProps & ViewProperties> {
+export class RNCamera extends Component<RNCameraProps > {
   static Constants: Constants;
 
   _cameraRef: null | NativeMethods;
